### DISCUSSION TITLE
How To Add Videos In Kade Engine 1.5.4 & Soon 1.6? (HELP PLEASE IM GOING INSANE)

### DISCUSSION THREAD

**Discussion by ItsLJcool** [KadeArchive/Kade-Engine#1369] [Q&A]
Hi! So im trying to add cutscenes into fnf, but i have no experiance in coding, so i decided to look at code that does use videos and see how it works, ive been trying to do this for 2 days now, but i still have no idea how to add webm, mp4, etc. videos Before, after a song, or just in the TitleState, I used Vs Nonsense Code to figure out how to add video's / cutscenes. But im still lost

Please help me D:

(Yes ive looked at Brights Tutorial, but that is a pluggin, i want to know how to use the built in one in kade engine) and i can't see crap when looking at the code in his video

**Comment by Riskerman21** [comment]
try this instead:
[https://www.youtube.com/watch?v=2B7dqNB6GcE](https://www.youtube.com/watch?v=2B7dqNB6GcE)
